e2e/specs/drug-orders.spec.ts

```diff
@@ -222,7 +143,38 @@ test('Record, edit and discontinue a drug order', async ({ page }) => {
   });
 });
 
+test('Cancel a existing drug order', async ({ page, api }) => {
+  url = process.env.E2E_BASE_URL;
+  await test.step('When I click on the Orders section', async () => {
+    await page.goto(url + `/spa/patient/${patient.uuid}/chart/Orders`);
```

Comment:
This is actually a good place to use a custom page, so the we don't have to do this here.